### PR TITLE
Added column arg to get_grdc_data

### PR DESCRIPTION
**Pull request by sverhoeven**
Refs #190

### CODE REVIEW

File: ewatercycle/observation/grdc.py [hunk 1]
```diff
@@ -8,11 +8,13 @@
...
 logger = logging.getLogger(__name__)
 
+
 def get_grdc_data(station_id: str,
                   start_time: str,
                   end_time: str,
                   parameter: str = 'Q',
-                  data_home: str = None) -> Tuple[pd.core.frame.DataFrame, Dict[str, Union[str, int, float]]]:
+                  data_home: str = None,
+                  column: str ='streamflow') -> Tuple[pd.core.frame.DataFrame, Dict[str, Union[str, int, float]]]:
```
Comment (Peter9192) [inline ewatercycle/observation/grdc.py]:
```suggestion
                  column: str = 'streamflow') -> Tuple[pd.core.frame.DataFrame, Dict[str, Union[str, int, float]]]:
```

Comment (Peter9192) [reply ewatercycle/observation/grdc.py]: And the line seems very long..

File: ewatercycle/observation/grdc.py [hunk 2]
```diff
@@ -32,6 +34,7 @@ def get_grdc_data(station_id: str,
         data_home : optional. The directory where the daily grdc data is
             located. If left out will use the grdc_location in the eWaterCycle
             configuration file.
+        column: optional. Name of column in dataframe of streamflow.
```
Comment (Peter9192) [inline ewatercycle/observation/grdc.py]:
```suggestion
        column: optional. Name of column in dataframe. Default: "streamflow".
```

**Review by Peter9192 — CHANGES_REQUESTED** [review]
Small formatting issue. Should we also update the examples? Perhaps at least this one: https://ewatercycle.readthedocs.io/en/latest/user_guide.html#Observations

**Comment by sverhoeven** [comment]
Thanks for the review, I updated the user guide nb.

**Review by Peter9192 — APPROVED** [review]

**Pull request merged**